wsagent/che-core-api-languageserver/src/test/java/org/eclipse/che/api/languageserver/registry/LanguageServerRegistryImplTest.java

```diff
@@ -107,7 +113,7 @@ protected String extractProjectPath(String filePath) throws LanguageServerExcept
         .thenAnswer(invocation -> completedFuture(Pair.of(languageServer, initializeResult)));
   }
 
-  @Test
+  @Test(enabled = false)
```

Comment:
This PR is not marked as WIP as I can see. Can you elaborate whether you are going to merge this PR with a disabled test?

Reply:
This pull request has no `status/code-review` label as well, but you're still reviewing it. Of course I have no plans to merge disabled unit test as that will cause test suit regression, thanks.

Reply:
thanks for the clarification. BTW there is a bot which puts label `status/code-review` on PRs.